cmd/litestream/main.go

```diff
@@ -775,3 +795,38 @@ func (v *indexVar) Set(s string) error {
 	*v = indexVar(i)
 	return nil
 }
+
+func parseBucketInfo(c *ReplicaConfig) error {
+	if c.BucketInfo == "" {
+		return nil
+	}
+
+	f, err := os.Open(c.BucketInfo)
+	if err != nil {
+		return fmt.Errorf("unable to open BucketInfo file: %w", err)
+	}
+
+	bi := cosiapi.BucketInfo{}
+
+	if err := json.NewDecoder(f).Decode(&bi); err != nil {
+		return fmt.Errorf("unable to decode BucketInfo: %w", err)
+	}
+
+	switch {
```

Comment:
I think this is a cool PR, but this switch is not configuring Azure or GCP.

Reply:
Yep, it is not. I will do that soon for Azure, not sure about GCP, as there is no GCP secret defined in COSI alpha specification.

Reply:
Ah, I don't know if the Azure Blob provides enough info for the `litestream` to configure replica client - it is only giving the `BucketName` and `AccessToken` - this probably is not enough data.